source/blog/2018-08-03-the-ember-times-issue-58.md

```diff
@@ -22,8 +22,10 @@ Check out what's been going in Emberland 🐹 these days:
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [The Quest for Better Documentation](https://github.com/emberjs/website/pull/3481)
+There’s a [brand new RFC](https://github.com/jenweber/rfcs-1/blob/cli-guides/active/0000-cli-guides.md) for restructuring and refreshing the Ember CLI docs. [Jen Weber](https://twitter.com/jwwweber?lang=da) prepared a great overview of what needs to change to make **Ember CLI documentation great again**. This is especially important since these change could empower new contributors, correct outdated information and make new the learning process way easier.
 
+Besides a restructure of the information architecture Jen proposes to use personas as a way of understanding what changes are needed as well as transforming the site into an Ember app.
```

Comment:
Do you think it would be cool to add a call to action at the end of the section? It could be something along the lines of `Community feedback is as always appreciated. Head over to the [proposal](https://github.com/ember-cli/rfcs/pull/120) and leave your comments below`